### DISCUSSION TITLE
Who are the "stakeholders" or "funders" of the project

### DISCUSSION THREAD

**Discussion by yarikoptic** [nebari-dev/nebari#2585] [Q&A] [answered]
I am new to nebari. Upon a quick look I found no history or current statement on what/how development of nebari has started or what funds (if any) supported its development directly or indirectly (e.g. a feature developed by a company X or under a .gov funding Y).   E.g. I spotted that @yuvipanda was one of the first developers but I know that he is now part of the https://github.com/2i2c-org and [there are some hits about 2i2c in search here](https://github.com/search?q=org%3Anebari-dev+2i2c&type=issues), but not clear what are relationships between IMHO quite closely related (if not duplicating) efforts. Describing what motivated/motivates development of the nebari and relationships with related/similar projects would have been very much appreciated.

And for the sake of transparency, would be valuable what funds behind drive the development. E.g. for relevant to our academic field standardization effort we have a page where we list funding sources in https://bids.neuroimaging.io/acknowledgments.html . 

Likely might be a good addition to https://github.com/nebari-dev/governance/ collection of documents.

Thank you in advance for the clarifications!

**Comment by Adam-D-Lewis** [comment]
Hi @yarikoptic, most of the development of Nebari is done by staff at Quansight currently.  Quansight was founded by Travis Oliphant (creator of Numpy, Scipy, Numba, founder of Anaconda). You can find more info about Quansight at https://quansight.com/about-us/. 

Among other services, Quansight offers Nebari support services to clients which help fund the development of Nebari.
As for 2i2c, You may be interested in reading the answer to "How does Quansight and Nebari compare to 2i2c’s managed JupyterHub cloud service?" on https://quansight.com/nebari-services/.  I'll post it here for convenience.
>Quansight counts 2i2c as a friend organization. They implement a similar approach of providing services around open source tools. We both use the model of providing managed services on top of open source thereby avoiding vendor lock-in. Where 2i2c manages a JupyterHub installation, Quansight manages Nebari which provides JupyterHub plus powerful environment management, built-in scalable computing, visualization capability, workflow scheduling, AI/ML tools, and more.

>Where 2i2c primarily targets academic and research institutions, Quansight primarily targets companies and government agencies. Quansight will fully manage your Nebari installation in a way that you can always “take over” and manage it yourself. We want to help every organization enable their data-curious subject matter experts to build powerful models, dashboards, and applications easily while quickly taking advantage of the evolving landscape of powerful open-source tools.

We have a biweekly community meeting if you're interested in attending.  https://www.nebari.dev/docs/community/#community-meetings

**Reply by yarikoptic** [reply]
Thank you @Adam-D-Lewis !

**Comment by dharhas** [comment, marked as answer]
@yarikoptic 

I would say there have been a couple of funding sources over the lifetime of both Nebari (formerly QHub) and Conda-Store which is a fairly critical subsystem:

- Initially internal funding by Quansight since we needed it for our work and client projects
- Various Quansight clients wanted specific features and let us open source the work/contribute it back.
- Nebari Support/Training Contracts - https://quansight.com/nebari-services/
- Government Grants
  - Nebari: Metrostar & Quansight have a joint grant from DoD CDAO JATIC that has been funding a lot of recent development (https://blog.metrostar.com/news/open-source-ai-dod-jatic)
  - Conda-Store: STF Award (https://www.sovereigntechfund.de/tech/conda-store)     

(Clients in the past have also included Govt agencies like the USGS, DEVCOM, Universities etc, but mostly Industry folks)

Yuvi worked on the predecessor to Nebari (QHub) before 2i2c was formed. I would say the 2i2c and Nebari  are both opinionated "distributions" of JupyterHub with different focuses. At a high level I would say 2i2c is focussed more on a "2i2c managed" JupyterHub platform and Nebari is focused more on a "self-managed" JupyterHub platform. Plus of course the different target audiences - Academic vs Non-Academic. There is of course a decent overlap.
